tests/lib/tools/fs-state

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: fs-state wait-for-file [filename] [iterations] [sleep-time]"
+    echo "usage: fs-state mock-dir [dirname]"
```

Comment:
Usually only the first line uses the word "usage: ...", the rest should be indented without the repeated "usage" prefix.

Reply:
done